src/SDKs/Compute/Management.Compute/Generated/Models/Page1.cs

```diff
@@ -2,17 +2,17 @@
 // Licensed under the MIT License. See License.txt in the project root for
 // license information.
 //
-// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
@shahabhijeet  The file is generated by autorest.  I don't know why autorest creates these files.